templates/web/fixmystreet-uk-councils/about/privacy.html

```diff
@@ -8,6 +8,23 @@ <h1>Privacy, cookies, and third party services</h1>
 
 <h2>Privacy Policy</h2>
 
+[% IF c.cobrand.moniker == 'bathnes' %]
```

Comment:
Suppose this isn't worth breaking out into its own template for a single council, but let's do it if another council asks.